src/analysis/normed/group/basic.lean

```diff
@@ -1516,6 +1516,14 @@ by simp only [←dist_one_right, dist_pi_le_iff hr, pi.one_apply]
 @[to_additive pi_nnnorm_le_iff]
 lemma pi_nnnorm_le_iff' {r : ℝ≥0} : ∥x∥₊ ≤ r ↔ ∀ i, ∥x i∥₊ ≤ r := pi_norm_le_iff' r.coe_nonneg
 
+@[to_additive pi_norm_le_iff''] lemma pi_norm_le_iff''' [nonempty ι] : ∥f∥ ≤ r ↔ ∀ b, ∥f b∥ ≤ r :=
```

Comment:
How about `pi_norm_le_iff_of_nonempty'` instead of `'''`?

Reply:
Would you prefer renaming `pi_norm_le_iff` to `pi_norm_le_iff_of_nonneg`, or maybe do both?

Reply:
I'm happy with either option, would just be nice to avoid adding another `'''`!